tactics/docker.py

```diff
@@ -0,0 +1,22 @@
+from charmtools.build.tactics import WheelhouseTactic
+
+
+class DockerWheelhouseTactic(WheelhouseTactic):
+    def __call__(self, venv=None):
+        # this was due to some weird scoping issue during execution.
+        # investigate more later
+        from charmtools import utils
+        create_venv = venv is None
+        venv = venv or path(tempfile.mkdtemp())
```

Comment:
I'm not entirely sure, as I had to import utils 2 lines above that.

I do believe however that venv is being set before hand and being passed in. As temple is not included either and would throw an error.